src/js/Pages/SharePage.js

```diff
@@ -1,14 +1,10 @@
 import m from 'mithril'
 
 import { Box } from '../Components/Box'
-import {
-	LikeTile,
-	ReplyTile,
-	RSVPTile,
-	BookmarkTile
-} from '../Editors/Tiles'
+import Tiles from '../Editors/Tiles'
```

Comment:
Missing `Store` import here I think:

```suggestion
import Store from '../Models/Store'
import Tiles from '../Editors/Tiles'
```